charts/ai-service/tests/artifacts-cache._test.yaml

```diff
@@ -0,0 +1,36 @@
+suite: test artifacts cache template
+templates:
+  - templates/artifacts-cache.yaml
+
+tests:
+  - it: should create PVC when enabled
+    set:
+      artifactsCache:
+        enabled: true
+        name: "test-cache"
```

Comment:
if you make a testcase without that you might see it fail as the names disconnect